src/pcapi/routes/native/v1/settings.py

```diff
@@ -10,14 +10,25 @@
 @blueprint.native_v1.route("/settings", methods=["GET"])
 @spectree_serialize(api=blueprint.api, response_model=serializers.SettingsResponse)
 def get_settings() -> serializers.SettingsResponse:
-    current_deposit_version = conf.get_current_deposit_version()
+    feature_dict = get_features_dict(
+        [
+            FeatureToggle.ENABLE_NATIVE_APP_RECAPTCHA.name,
+            FeatureToggle.ALLOW_IDCHECK_REGISTRATION.name,
+            FeatureToggle.AUTO_ACTIVATE_DIGITAL_BOOKINGS.name,
+            FeatureToggle.ENABLE_NATIVE_ID_CHECK_VERSION.name,
+            FeatureToggle.ENABLE_PHONE_VALIDATION.name,
+            FeatureToggle.APPLY_BOOKING_LIMITS_V2.name,
```

Comment:
La signature de `get_features_dict()` dit qu'on doit lui passer une liste de `FeatureToggle`, pas une liste de string. Donc je pense qu'on doit plutôt faire ça :

```suggestion
            FeatureToggle.ENABLE_NATIVE_APP_RECAPTCHA,
            FeatureToggle.ALLOW_IDCHECK_REGISTRATION,
            FeatureToggle.AUTO_ACTIVATE_DIGITAL_BOOKINGS,
            FeatureToggle.ENABLE_NATIVE_ID_CHECK_VERSION,
            FeatureToggle.ENABLE_PHONE_VALIDATION,
            FeatureToggle.APPLY_BOOKING_LIMITS_V2,
```